emsdk.py

```diff
@@ -1440,6 +1440,54 @@ def emscripten_npm_install(tool, directory):
     errlog('Error running %s:\n%s' % (e.cmd, e.output))
     return False
 
+  # Manually install the appropriate native Closure Compiler package
+  # This is currently needed because npm ci will install the packages
+  # for Closure for all platforms, adding 180MB to the download size
+  # There are two problems here:
```

Comment:
Since `npm ci` is actually run above, perhaps mention either up there, or here in this comment that the above command deliberately excludes the native versions of closure compiler by the use of the `--no-optional` flag.